03_NSIDCDataAccess_Steiker/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Jupyter Notebook
+.ipynb_checkpoints
```

Comment:
You can safely remove this file since you already added this in the repositories `.gitignore` file above.

Reply:
@jomey Would it be better to leave the .gitignore file in the 03_NSIDCDataAccess_Steiker folder itself? Obviously I had some issues trying to remove the checkpoints folder and originally put the .gitignore file in the higherlevel repo but it may be best to keep it in my folder since that's the only folder I'm touching in my fork.

Reply:
I would keep it at the repository level since there are many notebooks in other folders. This way we can keep them out as a whole and others don't have to repeat the exercise you were doing (getting rid of them).
In general it is a good practice to have a `.gitignore` at the top level over subfolders.